cwr/parser.go

```diff
@@ -24,13 +24,19 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
+	"os"
 	"os/exec"
 )
 
 // ParseCWRFile parse a give cwr file and returns an array of registeredWorks
 func ParseCWRFile(cwrFileReader io.Reader, CWRDataApiPath string) (registeredWorks []*RegisteredWork, err error) {
 	//phase 1 : Transform cwr formatted file to cwr-json file using CWR-DataApi python script.
 	cmd := exec.Command("python3", CWRDataApiPath+"/cwr2json.py")
+	//Use explicit encoding because it seems like some of the CWR files are ISO-8859-1 encoded
```

Comment:
If only _some_ of the files are ISO-8859-1 encoded then don't we need to do some form of detection?

Reply:
We might need to do that. 
So far we do not have such a file (UTF-8 with a special chars included)  for a reference. 
Currently, we can parse all of the CWR sample files we got using that.
The problematic files are files which include non standard ascii chars (>127) ...when decoding that with latin1 (ISO-8859-1) all is ok.
The non problematic files just include chars in the normal ascii range (<127) so decoding that with latin1 (ISO-8859-1) does not make a difference.

Reply:
Ok let's revisit once we have tested this on more files.